scripts/build.sh

```diff
@@ -49,52 +49,6 @@ else
 fi
 
 #
-# run `npm install` to prepare the UI if it's changed since the last time we ran it
-#
-newest_commit=$(git rev-parse HEAD)
-last_commit_file="../.last_npm_installed_ui_commit"
```

Comment:
and we don't need npm install anymore. Latest UI changes remove that requirement.